tests/Extensions/Logging.UnitTests/OmexLoggerUnitTests.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Linq.Expressions;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Logging.Replayable;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.Omex.Extensions.Logging.UnitTests
+{
+	[TestClass]
+	public class OmexLoggerUnitTests
+	{
+		[TestMethod]
+		public void LogMessage_PropagatedToEventSource()
```

Comment:
The naming format of the unit tests isn't quite right throughout.

It should be `Method_WhenX_DoesY`, so here it would be something like `LogMessage_WithPropagatedEventSource_Succeeds`.

Applies throughout.

Reply:
It's not exactly correct here as far as I understand, this unit test checks that event source that was passed into constructor actually called to send a message, so `DoesY` is `MessagePropagatedToEventSource` in this case